lib/core/utils/css-parser.js

```diff
@@ -1,9 +1,17 @@
-import { CssSelectorParser } from 'css-selector-parser';
+import { createParser } from 'css-selector-parser';
 
-const parser = new CssSelectorParser();
-parser.registerSelectorPseudos('not');
-parser.registerSelectorPseudos('is');
-parser.registerNestingOperators('>');
-parser.registerAttrEqualityMods('^', '$', '*', '~');
+const parse = createParser({
+  syntax: {
+    pseudoClasses: {
+      definitions: {
+        Selector: ['is', 'not']
+      }
+    },
+    combinators: ['>'],
+    attributes: {
+      operators: ['^=', '$=', '*=', '~=']
+    }
+  }
+});
 
-export default parser;
+export default { parse };
```

Comment:
This changed the export to a named export so you'll also need to [update the import in matches](https://github.com/dequelabs/axe-core/blob/develop/lib/core/utils/matches.js#L1).